recipes/netpbm/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "netpbm" %}
+{% set version = "10.73.28" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/netpbm/files/super_stable/10.73.28/netpbm-10.73.28.tgz
+  fn: {{ name }}-{{ version }}.tgz
+  sha256: c0d32d4b9a53fde47dd9a36f45653e0ef51f6c390517f10b0c5056d1a8a844bd
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config  # [linux]
+    - flex
+    - gettext
+    - make
```

As you're using `sed` it's better to get it from conda-forge rather than the host:
```suggestion
    - make
    - sed
```
(The `-i` flag isn't supported by macOS's `sed` and it also makes it easier for someone to build the recipe locally if they want.)